.chloggen/resourcedetectionprocessor_openshift_bump_semconv.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Is this a breaking change? I would have expected this to be dealt with by schemas but I don't see a mention of it on https://github.com/open-telemetry/opentelemetry-specification/blob/v1.18.0/schemas/1.18.0

Reply:
hm... I did not see this file before. Its only mentioned in the [release of v1.18.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.18.0/CHANGELOG.md#semantic-conventions-1). It seems that should have happened similar to [this](https://github.com/open-telemetry/opentelemetry-specification/pull/2957/files).

How would you recommend to proceed? Can we add this info to `1.18.0` or do we need to create a `1.18.1` release?

Reply:
Let's ask on an issue on the specification repo whether this should have been in the `1.18.0` schema, if it should have been then I guess we would need 1.18.1, yes

Reply:
Based on the conversation on open-telemetry/opentelemetry-specification/issues/3282, I think we should mark this as a breaking change, since users have no graceful way of handling this

Reply:
Changed it :+1: